<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewEmployeeDialog</class>
 <widget class="QDialog" name="ViewEmployeeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="empPayrollTable">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>611</width>
     <height>191</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Start Date</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>End Date</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Work Units</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Salary Per Unit</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Total</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="backBtn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>460</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addBtn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>430</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Start Date:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="startDateTxt"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="exportBtn">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>50</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Export CSV</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>541</width>
     <height>18</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="empIdLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="empNameLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Pay Period:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="payPeriodLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Role:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="roleLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>390</y>
     <width>396</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Work Units:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="workUnitsTxt"/>
    </item>
    <item>
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Salary Per Unit:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="salaryPerUnitTxt"/>
    </item>
   </layout>
  </widget>
  <widget class="QCalendarWidget" name="startCalendar">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>301</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Display:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="displayCombo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>104</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Start Time:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTimeEdit" name="startTimeEdit"/>
    </item>
    <item>
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>End Time:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTimeEdit" name="endTimeEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="payBtn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>430</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Pay</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
